sources/web/datalab/polymer/components/file-browser/file-browser.ts

```diff
@@ -260,6 +260,12 @@ class FileBrowserElement extends Polymer.Element implements DatalabPageElement {
       }
     }
 
+    if (!fileId && this.fileManagerType === 'bigquery') {
```

Comment:
Is this the best place for this? Can we maybe set it in the data browser itself, or in the bigquery gapi module?
I'll approve to unblock in case you want to move forward with this.

Reply:
I admit this seems like a bit of a hack. I could instead add a method such as defaultStartLocation() to the FileManager interface, and make BigQueryFileManager return this location. Would that be better?

Reply:
How about adding this to the `_loadStartupPath` instead? That method should change to accommodate things like this anyway, because if we couldn't get the user's last saved location, we'll need a default there.

Reply:
Done.